### PR TITLE
Add aks initial config

### PR DESCRIPTION
**Pull request by pasqualedevita**
this pr add:
- aks storage account infra to store aks terraform state
- aks configuration by terraform (namespaces, configmaps, ingress)

how to apply changes:
- run `bash scripts/setup.sh DEV-CSTAR` to setup `.bastianhost.ini` file and download kubeconfig (repeat only if bastian host ip changes)
- run `bash terraform.sh apply|plan|destroy DEV-CSTAR` to run terraform

require:
- https://github.com/pagopa/azurerm/pull/15
- kubectl installed on local machine
- helm installed on local machine

### CODE REVIEW

File: src/aks/ingress.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# from Microsoft docs https://docs.microsoft.com/it-it/azure/aks/ingress-internal-ip
+resource "helm_release" "ingress" {
+  name       = "nginx-ingress"
+  repository = "https://kubernetes.github.io/ingress-nginx"
+  chart      = "ingress-nginx"
+  version    = "3.31.0"
+  namespace  = kubernetes_namespace.ingress.metadata[0].name
+
+  values = [
+    "${templatefile("${path.module}/ingress/loadbalancer.yaml", { load_balancer_ip = var.ingress_load_balancer_ip })}"
```
Comment (uolter) [inline src/aks/ingress.tf:10]: Well actually the naming convention suggests to name the template file ending with `.tpl`
Indeed it's not a valid yaml file.
Something like: loadbalancer.yaml.tpl

**Review by uolter — CHANGES_REQUESTED** [review]

File: src/aks/ingress.tf (hunk 1 above)
Comment (pasqualedevita) [reply src/aks/ingress.tf:10]: I can update to .tpl but changing extension we lost file formatting.
Is this convention mandatory?

**Review by uolter — APPROVED** [review]

File: src/aks/ingress.tf (hunk 1 above)
Comment (pasqualedevita) [reply src/aks/ingress.tf]: done in 27d98f28c2018277d187c9736bf20b9c8b704869

**Review by uolter — APPROVED** [review]

**Comment by pasqualedevita** [comment]
storage created

**Pull request merged**